kernel/bpf/verifier.c

```diff
@@ -762,6 +765,58 @@ static int unmark_stack_slots_dynptr(struct bpf_verifier_env *env, struct bpf_re
 	return 0;
 }
 
+static void __mark_reg_not_init(const struct bpf_verifier_env *env,
+				struct bpf_reg_state *reg);
+
```

Comment:
Seems silly but should we move it so its more inline with its final position, at this code point.

It would be after `static bool dynptr_type_refcounted(enum bpf_dynptr_type type)` but before `static int destroy_if_dynptr_stack_slot(struct bpf_verifier_env *env, struct bpf_func_state *state, int spi);` above.

Reply:
Not silly.  I like this idea and I have pushed that change.

Reply:
Looks good (the rev compare) https://github.com/ctrliq/kernel-src-tree/compare/42b376fd1dd693fbef85844f862fefb54eb31f92..75d514096d8a045d685a1385cef05d99e12144bf

:shipit: